tdrs-backend/tdpservice/search_indexes/test/test_model_mapping.py

```diff
@@ -0,0 +1,389 @@
+"""Tests for elasticsearch model mapping."""
+
+import pytest
+from faker import Faker
+from django.db.utils import IntegrityError
+from tdpservice.search_indexes.models import T1, T2, T3, T4, T5, T6, T7
```

Comment:
@jtimpe once the fields are updated per record type/model, these tests will need updating too.